### ISSUE TITLE
Git History Cleanup

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#4538]
# List of PRs not in a release

**Comment by cjcolvar** [comment]
PRs in 7.1.1:
4166
4163
4161
4157
4149
4148
4147
4137
4131
4126
4106
4105
4063
4060
4059
4047
4133
4132
4129
4127
4115
4112
4091
4088
4086
4065
4064
4058
4053
4160
4159
4156
4142
4110
4108
4101
4100
4099
4075
4074
4071
4143
4138
4134
4119
4102

**Comment by cjcolvar** [comment]
PRs in 7.2:
4239
4246
4248
4260
4271
4124
4085
4080
4104
4293
4309
4245
4253
4254
4281
4282
4294
4302
4202
4203
4216
4241
4250
4258
4261
4262
4290
4231
4237
4152
4168
4176
4188
4154
4175
4136
4125
4165
4169
4177
4181
4184
4186
4132
4213
4218
4240
4244
4283
4286
4296
4299
4301
4303
4223
4230
4187
4200
4201

**Comment by cjcolvar** [comment]
PRs in 7.2.1:
4443
4423
4439
4438
4437

**Comment by cjcolvar** [comment]
PRs in releases that aren't in the list of going into develop:
4047
4059
4060
4099
4105
4126
4132
4137
4147
4149
4157
4163

PRs to develop that haven't been in a release:
1876
4073
4151
4179
4204
4211
4289
4315
4316
4317
4320
4321
4323
4324
4335
4336
4338
4341
4342
4353
4354
4355
4356
4360
4364
4365
4366
4376
4379
4380
4381
4385
4386
4389
4390
4399
4407
4415
4422
4436
4450
4451
4459
4462
4464
4465
4468
4483
4484
4496
4498
4499
4500
4501
4502
4503
4504
4510
4514
4517
4525
4526
4534
4535
4537

**Comment by apertome** [comment]
I've reviewed the last section here; checked items are already in the `develop` branch and thus require no action.

The ones that I left unchecked need to be merged into `develop`, unless there is some reason not to (they all look relevant to me)

**Comment by Dananji** [comment]
I reviewed the second set of PRs.
1. We need a PR to remove the following files; [jumpforward.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/jumpforward.svg), [skipback.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/skipback.svg), and [seekmedia.css](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/seekmedia.css). These got into the `develop` branch when partially merging #4151. 
2. There are 3 GoogleDrive related PRs (#4360, #4364, #4381), and I'm not sure what needs to be done at the moment about these. When we bring in #4289 these 3 need to be brought in as well.

**Comment by Dananji** [comment]
One thing I noticed was that the `develop` branch has SAML in it, which was not in the old `develop` branch.

**Comment by cjcolvar** [comment]
@Dananji It has SAML in it because the SAML code is in `staging`.  Do you think it shouldn't be in `develop`?

**Comment by cjcolvar** [comment]
>     1. We need a PR to remove the following files; [jumpforward.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/jumpforward.svg), [skipback.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/skipback.svg), and [seekmedia.css](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/seekmedia.css). These got into the `develop` branch when partially merging [ME.js player keyboard support #4151](https://github.com/avalonmediasystem/avalon/pull/4151).

It sounds like we can skip #4204 then, right?

**Comment by cjcolvar** [comment]
I worked through the 3rd set of PRs.  
@joncameron Could you go through and figure out which unchecked ones we should bring over into the `develop` branch?  If it is helpful we could do this synchronously on a video call.

**Comment by Dananji** [comment]
> > ```
> > 1. We need a PR to remove the following files; [jumpforward.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/jumpforward.svg), [skipback.svg](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/skipback.svg), and [seekmedia.css](https://github.com/avalonmediasystem/avalon/blob/f27f13ab1268c23e85504ec2c22ab10df41b60ff/vendor/assets/stylesheets/mediaelement/plugins/seekmedia.css). These got into the `develop` branch when partially merging [ME.js player keyboard support #4151](https://github.com/avalonmediasystem/avalon/pull/4151).
> > ```
> 
> It sounds like we can skip #4204 then, right?

Yes we can skip it (I'll mark it in the list), and do the suggested cleanup after we resolve git history.

**Comment by Dananji** [comment]
> @Dananji It has SAML in it because the SAML code is in `staging`. Do you think it shouldn't be in `develop`?

I'm not sure where the SAML code should go. I noticed that it's in there when ran the code in my local dev environment. Maybe the best person to ask is @apertome ?

**Comment by joncameron** [comment]
Let's do a call and review through them together, that makes sense to me after starting to read through these.

**Comment by apertome** [comment]
I'm not sure where the SAML parts make sense. I agree we could use a more detailed discussion.
